### PR TITLE
Post-TTA migration clean up

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-4481](https://trello.com/c/iT26TLTj/4481-migrate-tta-service-to-the-git-website)

### Context

We recently migrated the core TTA sign up journey to the website. We did this with the least number of changes possible to make it safer/easier to port across. This PR looks at areas that can now be cleaned up.

### Changes proposed in this pull request

- Namespace CRM models

To avoid confusion I'm moving the CRM-related models into their own namespace.

- Update TTA sign up with GiT design

Minor tweaks to the TTA sign up to make it look and feel more like the GiT website.

- Fix CodeClimate warnings

Fixes a number of CodeClimate warnings that were flagged.

- Re-design TTA start page and consolidate layouts

There is a redesigned TTA start page we are now using (matching the mailing list sign up start page).

I have also opted to consolidate the form partials across the different sign up journeys to DRY things up.

- Update adviser sign up completion page

Updates the adviser sign up completion page to match the new design.

### Guidance to review

[Re-designed start page](https://review-get-into-teaching-app-3271.london.cloudapps.digital/teacher-training-adviser/sign_up/identity)
[Re-designed end page](https://review-get-into-teaching-app-3271.london.cloudapps.digital/teacher-training-adviser/sign_up/completed)

The sign up journey is still disabled in production at the moment. That being said, **the refactoring will effect the other sign up journeys, so we should double check them before shipping this**.

### CODE REVIEW

**Review by gazcarr — DISMISSED** [review]
Design looks good to me

File: app/views/teacher_training_adviser/steps/completed.html.erb [hunk 1]
```diff
@@ -1,64 +1,55 @@
...
+
+  <p>It may take us longer to respond to you because we're getting a lot of requests for advisers at the moment. We’ll be in touch as soon as we can. Thank you for your patience.</p>
+  <% end %>
+
+  <h2 class="heading-m">Understanding your options</h2>
+
+  <p>As well as an adviser, you can find help and guidance on the Get Into Teaching website. There, you can:</p>
+
+  <ul>
+    <li>discover the different <%= link_to "ways to train", "/train-to-be-a-teacher" %> to become a teacher</li>
```
Comment (gemmadallmandfe) [inline app/views/teacher_training_adviser/steps/completed.html.erb:36]: @ethax-ross All of the links in this section are opening halfway down the destination page (ie keeping the position on the page the same as on the completion page) - they need to open at the top of the destination page

**Comment by gemmadallmandfe** [comment]
@ethax-ross I've been through the **mailing list funnel**, the **callback funnel** and the **events funnel** and they all look fine to me - I've been through the different branching, verification and errors

I've also been through the **adviser funnel** on GIT - I left a couple of comments yesterday but if you want to address these on a separate PR, please do!

**Review by asatwal — APPROVED** [review]
👍

**Pull request merged**